test/unit/adapters/parcel-permissions-fetcher.spec.ts

```diff
@@ -0,0 +1,183 @@
+import { createLogComponent } from '@well-known-components/logger'
+import { createParcelPermissionsComponent } from '../../../src/adapters/parcel-permissions-fetcher'
+import { ISubgraphComponent } from '@well-known-components/thegraph-component'
+import { TheGraphComponent } from '../../../src/ports/the-graph'
+
+function createMockTheGraphComponent(mockLandQuery: jest.Mock): TheGraphComponent {
+  return {
+    start: async () => {},
+    stop: async () => {},
+    ethereumCollectionsSubgraph: {} as ISubgraphComponent,
+    maticCollectionsSubgraph: {} as ISubgraphComponent,
+    ensSubgraph: {} as ISubgraphComponent,
+    thirdPartyRegistrySubgraph: {} as ISubgraphComponent,
+    landSubgraph: {
+      query: mockLandQuery
+    } as ISubgraphComponent
+  }
+}
+
+describe('ParcelPermissionsComponent', () => {
+  it('returns correct permissions when user is parcel owner', async () => {
+    const logs = await createLogComponent({})
+    const userAddress = '0xuser'
+    const parcelX = 10
+    const parcelY = 20
+
+    const mockQuery = jest.fn().mockResolvedValue({
+      parcels: [
+        {
+          x: parcelX.toString(),
+          y: parcelY.toString(),
+          owner: {
+            address: userAddress
+          },
+          updateOperator: '0xsomeoneelse'
+        }
+      ],
+      estates: []
+    } as unknown)
+
+    const theGraph = createMockTheGraphComponent(mockQuery)
+
+    const component = await createParcelPermissionsComponent({ theGraph, logs })
+    const permissions = await component.getParcelPermissions(userAddress, parcelX, parcelY)
+
+    expect(permissions).toEqual({ address: true, operator: false })
+    expect(mockQuery).toHaveBeenCalledWith(expect.any(String), {
+      addressLower: userAddress,
+      x: parcelX,
+      y: parcelY
+    })
+  })
+
+  it('returns correct permissions when user is parcel operator', async () => {
```

Comment:
What do you think about writing the it as:
`resolves to the operator permission being true when user is parcel owner`
Something similar would apply to other `it`s.